param/__init__.py

```diff
@@ -1201,6 +1217,17 @@ def get_range(self):
         raise NotImplementedError("get_range() must be implemented in subclasses.")
 
 
+def _update_selector_default(p):
+    return []
+
+
+def _update_selector_check_on_set(p):
+    if len(p.objects) == 0:
+        return False
+    else:
+        return True
```

Comment:
```suggestion
def _update_selector_check_on_set(p):
    return (len(p.objects) != 0)
```

Also maybe rename `_compute_selector_checking_default`?